samples/sensor/die_temp_polling/boards/esp32c3_devkitm.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
```

Comment:
Shouldn't this use your own company's copyright @hirokin94?

Reply:
@carlescufi Thank you for your review.
I fixed the copyright.